### PR TITLE
feat: drawers added

### PR DESCRIPTION
**Pull request by sonishreyas**
## Change Logs

Feature: drawers added

## Screenshot
image

##  Preview Link
https://deploy-preview-18--ripple-ui.netlify.app/components/drawers/drawers.html

### CODE REVIEW

**Review by rishmi5h — COMMENTED** [review]
looks good and functional.

File: components/drawers/drawers.html [hunk 1]
```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Shreyas c. Soni">
+    <!-- <link rel="stylesheet" href="https://ripple-ui.netlify.app/css/components.css" /> -->
```
Comment (Aditi0505) [inline components/drawers/drawers.html:9]: Suggestion: Can remove unnecessary comments.

**Review by Aditi0505 — APPROVED** [review]
Looks good to me

File: components/drawers/drawers.html (hunk 1 above)
Comment (sonishreyas) [reply components/drawers/drawers.html:9]: Thanks I will remove that once all my components CSS is in dev

**Pull request merged**